root/scripts/recipepage.js

```diff
@@ -142,6 +142,14 @@ async function init() {
   } else {
     button.style = 'color:grey';
   }
+
+  (function(d, s, id) {
+    var js, fjs = d.getElementsByTagName(s)[0];
+    if (d.getElementById(id)) return;
+    js = d.createElement(s); js.id = id;
+    js.src = "https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v3.0";
+    fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group34/rocketrecipes/pullRequest?prid=8487889)
```suggestion
  }(document, 'script', 'facebook-jssdk'));
```